llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -579,6 +537,62 @@ inline constexpr CpuInfo CpuInfos[] = {
                                AArch64::AEK_SSBS, AArch64::AEK_CSSC})},
 };
 
+struct ExtensionSet {
```

Comment:
This has to move after the definition of `ArchInfo` because the method `toLLVMFeatureList` needs to know about it. Since it is a template function its definitions ought to move in the header file, alongside its class definition.